### PR TITLE
Document and test all supported Python versions

### PR DESCRIPTION
**Pull request by jdufresne**
Document support for Python 3.8 (released October 14th, 2019) and PyPy
and add them to the test matrix.

### CODE REVIEW

**Comment by birkenfeld** [comment]
Thanks for porting to Git!

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [2.7, 3.5, 3.6, 3.7]
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, pypy, pypy3]
```
Comment (birkenfeld) [inline .github/workflows/build.yaml]: Looks like this should be "pypy2"?

**Comment by jdufresne** [comment]
Thanks, fixed pypy2.

**Comment by Anteru** [comment]
Not too impressed by the CI speed here, but running on PyPy is surely a good thing. Thanks for the PR!

**Pull request merged**